pkg/api/registry/registry.proto

```diff
@@ -22,61 +22,43 @@ message Destination {
     uint32 weight = 2;
 }
 
-message NetworkServiceManager {
-    string name = 1;
-    string url = 2;
-    google.protobuf.Timestamp expiration_time = 3;
-    string state = 4;
-}
-
 message NetworkServiceEndpoint {
```

Comment:
NetworkServiceKind is missing here.

Reply:
Not sure that it is required for structure NetworkServiceEndpoint, because we can easily handle search case by kind in discovery service.

Reply:
ok, how you plan to register Endpoint or manager?

Reply:
I would suggest using some network service name convention. For example for forwarders something like `xconn-network-service`, for managers `nsmgr-network-service`.

Reply:
I also don't mind to add 'kind' field for `NetworkServiceEndpoint`, but I would try to minimalize field count :)

Reply:
@haiodo What do you mean by NetworkServiceKind ?

Reply:
I'm a bit nervous about baking the 'Kind's in like that into the API.  NSMgrs are a concept in K8s... they might not be in the same way in a different environment (same with Forwarders).  The 'Kinds' listed:

* CROSS_CONNECT_NSE - feels like its just a Network Service, and could be handled as such.
* NETWORK_SERVICE_MANAGER - when/how do you envision us registering an NSMgr?  What Network Service would we register it as providing?
* NETWORK_SERVICE_ENDPOINT

I'm trying to wrap my head around the ways you think we would use these :)

Reply:
@edwarnicke NOTE: Kind currently used only in discovery service. 

Do you have any idea how we can discover only network service endpoints(i.e. we need to exclude managers, cross-connect nses from response entries)

Reply:
I'll turn that question around:

Give me a query that you are concerned would returns a NSMgr or a Cross-Connect NSE incorrectly?